### PR TITLE
this is the user profile feature

### PR DESCRIPTION
**Pull request by arpadhani**
please @liang review my code

### CODE REVIEW

File: third.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+my third commit
```
Comment (arpadhani) [inline third.txt]: pls add 4 spaces

**Comment by arpadhani** [comment]
👍

**Comment by Liang** [comment]
hmm. are you looking for different developer with name in "liang"?
I didn't remember I work on this project.
However, the line is simple and approved.

**Comment by arpadhani** [comment]
Hi, we apologize about that. Tagging feature was not working so I guessed username.

> On Apr 27, 2016, at 8:29 PM, Liang <notifications@github.com> wrote:
> 
> hmm. are you looking for different developer with name in "liang"?
> I didn't remember I work on this project.
> However, the line is simple and approved.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub <https://github.com/arpadhani/test/pull/1#issuecomment-215300272>

Adish R. Padhani
adishraza@gmail.com

**Pull request merged**